setup.py

```diff
@@ -33,6 +33,7 @@
       namespace_packages=['slimta'],
       install_requires=['gevent >= 1.1rc',
                         'pysasl >= 0.4.0',
+                        'dkimpy >= 0.9.0',
```

Comment:
I'd like to keep this dependency as optional. A few things:

Please move this to an `extras_require` parameter:

```python
extras_require={'dkim': ['dkimpy >= 0.9.0']},
```

And then move your new policy class to its own module (maybe `slimta.policy.dkimheader`). The test can stay put, just add `dkimpy >= 0.9.0` as a new line in `test/requirements.txt`.